### PR TITLE
fix(app) not updated email subject

### PR DESCRIPTION
**Pull request by blackwolf3342**
- Add LBS version in readme
- Fix delay for "Open in Getaccept"
- Fix not updated email subject and message
- Add skip template fields page if template not have they
- Add fetch company data related to deal
- Add picker for template roles
- Add folders to templates list

### CODE REVIEW

File: GetAccept-v2/app.js [hunk 1]
```diff
@@ -742,39 +779,73 @@
...
-            $.each(eval('viewModel.' + className), function (fieldName, fieldValue) {
-                var fieldData = new mapAvailableField(fieldName, fieldValue);
-                viewModel.availableLimeFields.push(fieldData);
-            });
+            $.each(viewModel.releatedModels, function (index , relatedClassName) {
+                if(!eval('viewModel.'+relatedClassName)) {
+                    return;
+                }
+                $.each(eval('viewModel.' + relatedClassName), function (fieldName, fieldValue) {
+                    if (fieldName == 'id' || fieldName == 'lymetype') {
```
Comment (andreasastrom) [inline GetAccept-v2/app.js:833]: `==` --> `===`to make it exact

File: GetAccept-v2/app.js [hunk 2]
```diff
@@ -742,39 +779,73 @@
...
         }
 
+        function getTemplateRoles() {
+            viewModel.templateRoles.removeAll();
+            apiRequest("templates/" + viewModel.selectedTemplateId() + '/roles', "GET", "", function (data) {
+                if (data) {
+                    $.each(data.roles, function (index, rolesData) {
+                        try {
+                            viewModel.templateRoles.push(rolesData);
+                        } catch (e) {
```
Comment (andreasastrom) [inline GetAccept-v2/app.js:816]: maybe log something more than just error?

Comment (blackwolf3342) [reply GetAccept-v2/app.js:816]: I think i can remove that

File: GetAccept-v2/app.js [hunk 3]
```diff
@@ -718,6 +731,30 @@
             });
         }
 
+        function getTemplateFolders(folderId = '') {
+            viewModel.templateFolders.removeAll();
+            let folderUrl = '';
```
Comment (andreasastrom) [inline GetAccept-v2/app.js:736]: const folderUrl = folderId? /${folderId} : ''

File: GetAccept-v2/app.js [hunk 4]
```diff
@@ -813,6 +884,18 @@
             return template;
         }
 
+        function templateFolder(templateFolder) {
+            var folder = this;
+            folder.name = templateFolder.name;
```
Comment (andreasastrom) [inline GetAccept-v2/app.js:885]: `folder = {...folder, name:  templateFolder.name, id: templateFolder.id}`

Comment (andreasastrom) [reply GetAccept-v2/app.js:885]: etc

File: GetAccept-v2/app.js [hunk 5]
```diff
@@ -849,6 +932,20 @@
             viewModel.showPersons(true);
         }
 
+        function showRecipientsList(index, role, event) {
+            viewModel.recipientsList().map((recipient) => {
+                if(recipient.roleId && recipient.roleId == role.role_id) {
```
Comment (andreasastrom) [inline GetAccept-v2/app.js:933]: ===

File: GetAccept-v2/app.js [hunk 6]
```diff
@@ -1447,7 +1557,7 @@
                         var sso_url = 'https://app.getaccept.com/auth/sso/login?token=' + escape(accessToken) + '&entity_id=' + entityId + '&go=' + escape(docUrl);
                         lbs.common.executeVba("GetAccept.OpenGALink", sso_url);
                         done();
-                    }, 3000);
+                    }, 11*1000);
```
Comment (andreasastrom) [inline GetAccept-v2/app.js:1556]: consider variable to make it more readable. Why 11*1000?

File: GetAccept-v2/app.js [hunk 7]
```diff
@@ -813,6 +880,18 @@
             return template;
         }
 
+        function templateFolder(templateFolder) {
+            var folder = this;
```
Comment (andreasastrom) [inline GetAccept-v2/app.js:884]: if you make this changes could you use a const as well

Comment (andreasastrom) [reply GetAccept-v2/app.js:884]: and return directly

File: GetAccept-v2/app.js (hunk 6 above)
Comment (blackwolf3342) [reply GetAccept-v2/app.js:1556]: I've tested it, and got 11 seconds for importing document and open in getaccept with ready document.
Described  [here](https://app.shortcut.com/getaccept/story/24248/lime-crm-add-delay-for-open-in-getaccept-when-uploading-file)

Comment (andreasastrom) [reply GetAccept-v2/app.js:1557]: can't we call the variable? `sleepWhileImporting` Or something then?

File: GetAccept-v2/app.html [hunk 8]
```diff
@@ -279,12 +293,36 @@ <h4 data-bind="click: toggleFileContainer(false)">
             <div class="ga-document ga-sendout-workflow">
                 <a class="ga-back" data-bind="text: localize.GetAccept.BACK, click: backDocument, icon: 'fa-arrow-left'"></a>
                 <div class="ga-document-fields-container">
+
+                    <!-- ko if: templateRoles().length -->
+                    <h4>Template Roles</h4>
```
Comment (andreasastrom) [inline GetAccept-v2/app.html:298]: Add translation

File: GetAccept-v2/app.html [hunk 9]
```diff
@@ -279,12 +293,36 @@ <h4 data-bind="click: toggleFileContainer(false)">
...
+                                        <span data-bind="text: name, tooltip: email, icon: 'fa-plus'"></span>
+                                    </li>
+                                </ul>
+                            </div>
+                            <!-- /ko -->
+                        </li>
+                    </ul>
+                    <!-- /ko -->
+                    <!-- ko if: selectedTemplateFields().length -->
+                    <h4>Template Fields</h4>
```
Comment (andreasastrom) [inline GetAccept-v2/app.html:318]: Add translation

File: GetAccept-v2/app.js [hunk 10]
```diff
@@ -742,39 +776,69 @@
...
-            $.each(eval('viewModel.' + className), function (fieldName, fieldValue) {
-                var fieldData = new mapAvailableField(fieldName, fieldValue);
-                viewModel.availableLimeFields.push(fieldData);
-            });
+            $.each(viewModel.releatedModels, function (index , relatedClassName) {
+                if(!eval('viewModel.'+relatedClassName)) {
+                    return;
+                }
+                $.each(eval('viewModel.' + relatedClassName), function (fieldName, fieldValue) {
+                    if (fieldName === 'id' || fieldName === 'lymetype') {
```
Comment (andreasastrom) [inline GetAccept-v2/app.js:826]: just to check. Is `lymetype` correct? Or should it be `limetype`

File: GetAccept-v2/app.js [hunk 11]
```diff
@@ -813,6 +878,18 @@
             return template;
         }
 
+        function templateFolder(templateFolder) {
+            var folder = this;
+            folder.name = templateFolder.name;
+            folder.id = templateFolder.id;
```
Comment (blackwolf3342) [inline GetAccept-v2/app.js:884]: Should be like this, because getting error `Uncaught SyntaxError: Invalid left-hand side in assignment `

**Review by andreasastrom — APPROVED** [review]

**Pull request merged**